content/docs/rendering-elements.md

```diff
@@ -1,75 +1,76 @@
 ---
 id: rendering-elements
-title: Rendering Elements
+title: Việc render các element
 permalink: docs/rendering-elements.html
 redirect_from:
   - "docs/displaying-data.html"
 prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
+Element là những khối xây dựng nhỏ nhất của các ứng dụng React.
```

Comment:
"những khối xây dựng nhỏ nhất" em thấy hơi kì chỗ này, nhưng cũng k biết phải dịch như thế nào cho dễ hình dung

Reply:
ù chỗ này khó quá, Chị cũng ko biết dịch sao.

Reply:
em được gợi ý từ một anh ngồi kế bên thì là: "Element là những mảnh ghép nhỏ nhất của các ứng dụng React."

Reply:
Thank you em. just updated.